change/@ni-nimble-components-77825b10-d033-43ac-9a4c-aa28738727ea.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
There was a slight change to the Chromatic snapshots. I'm guessing this is because of the `inline` change? It seems acceptable to approve but you should probably wait to do so until you decide whether to keep that as part of this PR; if you remove that change then maybe the diff will go away.

Reply:
Removed the `inline` as we found an issue in the edit comment in SLE. I created a bug and will resolve it in a follow-up PR. https://github.com/ni/nimble/pull/1709#discussion_r1427980861